src/libs/ReportUtils.ts

```diff
@@ -1151,7 +1151,7 @@ function hasOnlyTransactionsWithPendingRoutes(iouReportID: string | undefined):
  * If the report is a thread and has a chat type set, it is a workspace chat.
  */
 function isWorkspaceThread(report: OnyxEntry<Report>): boolean {
-    return isThread(report) && isChatReport(report) && !isDM(report);
+    return isChatReport(report) && !isDM(report);
 }
```

Comment:
Before a report was considered a workspace thread if it is a thread and it's a chat report, and it's not a DM.

Now, a report is considered a workspace thread if it's a chat report and not a DM. That doesn't make sense though, because a top level workspace chat like #announce is a chat report with a chat type, but it should not be considered a workspace thread.

Breaking it down:
isWorkspaceThread will be true if isChatReport is true, and isDM is false. isDM can be false if any of the pieces of the and statement are false. If it has a chat type, then !getChatType(report) is false. A #admins room has a chat type of "policyAdmins". Therefore isWorkspaceThread is true when it should not be, because a report is a thread if it has a parent report, and the #admins report does not.

![image](https://github.com/Expensify/App/assets/26260477/8ac07ff6-d185-4a4f-8341-16bf7d6bc24f)

Reply:
@neil-marcellini That makes sense! Thanks for pointing that out. Made the change here: https://github.com/Expensify/App/pull/37030/commits/6712b4589d94c2da01bbf157c388f57ff619238e
I'm back in my workspace now, so now can work on this PR normally (no more delays)